imagesloaded/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for imagesLoaded 3.1.8
+// Type definitions for imagesLoaded 4.1.1
```

Comment:
Thanks for the info! I'll try adding one next time I make changes to a package that's missing it.